ras_party/controllers/party_controller.py

```diff
@@ -21,8 +21,10 @@ def parties_post(party_data, session):
     :param party_data: packet containing the data to post
     :type party_data: JSON data maching the schema described in schemas/party_schema.json
     """
+    logger.info("party data in controller", party_data=party_data)
     errors = Business.validate(party_data, current_app.config['PARTY_SCHEMA'])
     if errors:
+        logger.info("errors", errors=errors)
```

Comment:
This seems near-identical to the line below - we should have one or the other

Reply:
Agree. Feels like debugging code that's been left in